src/main/java/seedu/address/logic/parser/Parser.java

```diff
@@ -177,16 +190,26 @@ private Command prepareSelect(String args) {
      * @return the prepared command
      */
     private Command prepareFind(String args) {
```

Comment:
As ive abstracted out the construction of the keyword list. It could be useful for future commands using keywords.